### DISCUSSION TITLE
Janitor Quality of Life

### DISCUSSION THREAD

**Discussion by ghost** [unitystation/unitystation#8596] [Ideas]
As Janitor, a few things make your life unnecessarily hard. I intend to list these here..

1. Ability to dump entire trashbags/ trashbags of holding into disposals by left-clicking it.

2. Stacking bullets. Please. I beg you.

3. Not having disposals access. (Not sure how universal this is but on some maps i cant walk out of disposals after being trashed, which is annoying since I am the trash man, I take out the garbage.

4. Trash bag too small. Seen a lot of people complain about this, I don't really agree since i think it may become too OP, but I think that the trash bag of holding definetly needs a fat size increase.

Cheers, Juke (On behalf of all janitors)

**Comment by NoooneyDude** [comment]
Related to #4938.

**Comment by PerfectTangent** [comment]
Perhaps the trashbags should have the ability to pick up all items on a square like the ore bag. That can fix the issue of piles of bullet casings.

I think the trashbag holds 15 items and the Of Holding variety can take 60. a 25% increase to both seems a reasonably conservative buff.

**Comment by Akbadain29** [comment]
The trashbag does have the ability to pick up all the items on one square like the ore bag. Just click on the bag icon up top when its in your hand to change the pick up setting from one-item to the-whole-tile.
What really needs to change is giving the bag icon up top the ability to switch the trashbag between pick-up-one, pick-up-all and dump-all.

**Comment by jamessimo** [comment]
I Recently in the discord we talked about making trash bags disposable. The idea is, a Janitor can have a stack of say 50 trash bags (like you would in real life) and when one is filled up they just dump the whole thing and tear off a new bag.

**Comment by Akbadain29** [comment]
I'm looking into doing that, I just have to make a trash bag roll item prefab with a stackable component and adapt the FoldedBodyBag script so you then open up the trash bag from it's folded state to the existing prefab.
